src/Goteo/Controller/TestController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/*
+* This file is part of the Goteo Package.
+*
+* (c) Platoniq y Fundación Goteo <fundacion@goteo.org>
+*
+* For the full copyright and license information, please view the README.md
+* and LICENSE files that was distributed with this source code.
+*/
+
+namespace Goteo\Controller;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class TestController extends BaseSymfonyController {
```

Comment:
This controller should be removed before accepting the PR. It's just to ease the testing of it.